docs/platforms/unreal/configuration/debug-symbols/index.mdx

```diff
@@ -7,7 +7,7 @@ Sentry requires [debug information files](/platforms/unreal/data-management/debu
 
 <Note>
 
-For Android, debug symbols upload is handled by [Sentry Android Gradle Plugin](/platforms/android/configuration/gradle).
+For Android, debug symbols upload is handled by [Sentry Android Gradle Plugin](/platforms/android/configuration/gradle). By default, it bumps Gradle version to 7.5 automatically however one can fallback to using its legacy version in case compatibility issues with other third-party plugins arise.
```

Comment:
```suggestion
For Android, debug symbols upload is handled by [Sentry Android Gradle Plugin](/platforms/android/configuration/gradle). By default, it bumps Gradle version to 7.5 automatically, however one can fallback to using its legacy version in case compatibility issues with other third-party plugins arise.
```